From b29bbcb859653be148ea7a94c804aafab5845a32 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 25 Sep 2017 20:59:59 +0200
Subject: [PATCH] build: cure some warnings

Chromium quality degrades as Google/Chromium.org exclusively
uses clang....

../../third_party/WebKit/Source/platform/wtf/text/TextCodec.h:76:52: warning: enum constant in boolean context [-Wint-in-bool-context]
 static_assert(kDataEOF, "DataEOF should be truthy");
---
 third_party/WebKit/Source/platform/wtf/text/TextCodec.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/WebKit/Source/platform/wtf/text/TextCodec.h b/third_party/WebKit/Source/platform/wtf/text/TextCodec.h
index 9c23de8d4ad1..20e878aea464 100644
--- a/third_party/WebKit/Source/platform/wtf/text/TextCodec.h
+++ b/third_party/WebKit/Source/platform/wtf/text/TextCodec.h
@@ -69,8 +69,8 @@ enum FlushBehavior {
 };
 
 static_assert(!kDoNotFlush, "DoNotFlush should be falsy");
-static_assert(kFetchEOF, "FetchEOF should be truthy");
-static_assert(kDataEOF, "DataEOF should be truthy");
+static_assert(kFetchEOF != 0, "FetchEOF should be truthy");
+static_assert(kDataEOF != 0, "DataEOF should be truthy");
 
 class WTF_EXPORT TextCodec {
   USING_FAST_MALLOC(TextCodec);
-- 
2.14.3

